icepyx/core/APIformatting.py

```diff
@@ -333,22 +334,14 @@ def check_values(self):
             self.partype != "required"
         ), "You cannot call this function for your parameter type"
 
-        default_keys = self.poss_keys["default"]
         spatial_keys = self.poss_keys["spatial"]
 
-        if all(keys in self._fmted_keys.keys() for keys in default_keys):
-            assert all(
-                self.fmted_keys.get(key, -9999) != -9999 for key in default_keys
+        # not the most robust check, but better than nothing...
+        if any(keys in self._fmted_keys.keys() for keys in spatial_keys):
+            assert any(
+                self.fmted_keys.get(key, -9999) != -9999 for key in spatial_keys
             ), "One of your formated parameters is missing a value"
-
-            # not the most robust check, but better than nothing...
-            if any(keys in self._fmted_keys.keys() for keys in spatial_keys):
-                assert any(
-                    self.fmted_keys.get(key, -9999) != -9999 for key in default_keys
-                ), "One of your formated parameters is missing a value"
-                return True
-            else:
-                return False
+            return True
```

Comment:
> Check Warning for this line in the list of commits.

The warnings come from the Codecov check (which got updated to work again). Codecov looks at your tests and your code and identifies lines of code that haven't been tested. The way it integrates with GH, it will call out any "new" line of code that isn't tested. Thus, whenever a line of code that is not covered by a test is updated, it considers it "new" and issues that warning. In an ideal world we'd have everything covered by tests, but for cases where it's not a matter of updating an existing test (i.e. there is currently no test), I've been ignoring the checks for now.